ci/docker/runtime_functions.sh

```diff
@@ -457,6 +457,24 @@ build_ubuntu_gpu_mkldnn() {
     report_ccache_usage
 }
 
+build_ubuntu_gpu_mkldnn_nocudnn() {
+    set -ex
+
+    build_ccache_wrappers
+
+    make  \
+        DEV=1                         \
+        USE_CPP_PACKAGE=1             \
```

Comment:
No need for the cpp package here since we don't run the cpp tests

Reply:
not sure about this . shouldnt we test if it builds fine with CUDNN with CPP_PACKAGE just in case cpp package depends on the flag to use cudnn specific functionality (unlikely) ?

Reply:
The CPP package just generates wrappers around the C-API and should not rely on backend-specific things like CUDA or CUDNN. Thus, I'd say that we should keep it separate and only build the CPP package if we're actually going to run it (which we do in another test)